components/scream/src/share/io/scream_output_manager.cpp

```diff
@@ -215,16 +209,20 @@ void OutputManager::run(const util::TimeStamp& timestamp)
 
       // Register time as a variable.
       auto time_units="days since " + m_case_t0.get_date_string() + " " + m_case_t0.get_time_string();
-      register_variable(filename,"time","time",time_units,1,{"time"},  PIO_REAL,"time");
+      register_variable(filename,"time","time",time_units,1,{"time"}, scorpio::nctype("double"),"time");
```

Comment:
Is this set to only be double for restarts? I can't tell from the snippet here. I'm reviewing on my phone...

Reply:
I kept time as a double all the time, since it's a scalar, so turning it to float doesn't really make a dent anyways...